src/main/lesson16/Main.java

```diff
@@ -0,0 +1,46 @@
+package main.lesson16;
+
+public class Main {
+    static BankApplication app = new BankApplication();
+
+    final static String EXCEPTION = "java.lang.Exception";
+    //WrongAccountException.class.getCanonicalName();
+
+    final static String ACCOUNT_EXCEPTION = "main.lesson16.exceptions.WrongAccountException";
+    final static String CURRENCY_EXCEPTION = "main.lesson16.exceptions.WrongCurrencyException";
+    final static String OPERATION_EXCEPTION = "main.lesson16.exceptions.WrongOperationException";
+
+    public static void processWrapper(String accountId, int amount, String currency) {
+
+        try {
+            app.process(accountId, amount, currency);
+        } catch (Exception e) {
+            switch (e.getClass().getCanonicalName()) {
+                case ACCOUNT_EXCEPTION:
```

Comment:
Instead of switch-case statement add multiple catch statements for each type of exception. Please do not complicate task